File: Server/Server/Models/Inside/CompletedQuiz.cs
namespace Server.Models.Inside
{
    public class CompletedQuiz
    {
        public int Id { get; set; }
        public int Score { get; set; }
        public int UserId { get; set; }
        public int QuizId { get; set; }
        public User User { get; set; } = null!;
        public Quiz Quiz { get; set; } = null!;
        public virtual ICollection<SelectedAnswer> SelectedAnswers { get; set; } = null!;
    }
}

File: Client/src/app/models/completedQuiz.ts

import { Quiz } from "./quiz";
import { SelectedAnswer } from "./selectedAnswer";
import User from "./user";

export class CompletedQuiz {
    id: number = -1;
    score: number = -1;
    userId: number = -1;
    quizId: number = -1;
    user!: User;
    quiz!: Quiz;
    selectedAnswers: Array<SelectedAnswer> = [];
    message: string = "";
}